src/main/scala/com/mowczare/kafka/streams/pds/model/model.scala

```diff
@@ -0,0 +1,6 @@
+package com.mowczare.kafka.streams.pds.model
+
+import com.avsystem.commons.serialization.HasGenCodec
+
+case class UniqueCount(count: Long)
```

Comment:
This looks weird. Is it model? Is it a part of user interface?

Reply:
It is a part of user interface. 
Hll and theta sketches returns unique count. To distinguish it from simple count I introduced UniqueCount type. In linkyfi, we had problems with ambiguity of long, as unique/simple count result. For example: monoid for unique count is not a good idea, while for simple count it is.

Reply:
name of the file ^

Reply:
I find it convenient to place all ADT in one place . This could be package object, but intellij sucks in finding classes from package objects.